README.md

```diff
@@ -114,10 +119,16 @@ For your convenience, Squiss provides direct access to the AWS SDK's SQS object,
 
 ## Events
 
+### deleted {Message}
+Emitted when a message is confirmed as being successfully deleted from the queue. The `handled` and `delQueued` events will also be fired for deleted messages, but that will come earlier, when the delete function is initially called.
+
 ### delError {Object}
 A `delError` is emitted when AWS reports that any of the deleted messages failed to actually delete. The
 object handed to you in this event is the AWS failure object described in the [SQS deleteMessageBatch documentation](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SQS.html#getQueueUrl-property).
 
+### delQueued {Message}
+Emitted when a message is queued for deletion, even if delete queuing has been turned off.
```

Comment:
If delete queuing has been turned off, then does the delete happen immediately? -- and the delQueued event just gets emitted regardless